src/module/functionality.js

```diff
@@ -0,0 +1,38 @@
+import Task from './task.js';
+import Storage from './localstorage.js';
+
+let taskList;
+
+if (Storage.getLocalStorage() === null) {
+  taskList = [];
+} else {
+  taskList = Storage.getLocalStorage();
+}
+
+export default class Features {
+  static addTaskList = (task) => {
+    if (task) {
+      const newTask = new Task(task);
+      taskList.push(newTask);
+      taskList.forEach((element, number) => {
+        element.index = number + 1;
+      });
+      Storage.saveLocalStorage(taskList);
+    }
+  }
+
+  static removeTask = (index) => {
+    taskList = Storage.getLocalStorage();
+    taskList.splice(index, 1);
+    taskList.forEach((task, index) => {
+      task.index = index + 1;
+    });
+    Storage.saveLocalStorage(taskList);
+  }
```

Comment:
- [ ] Kindly check [this loom video](https://www.loom.com/share/cc25722475a4420c8a62bab62f5777d4) and fix the bug in your to-do list and make sure, it is working correctly.